src/pages/Wishlist/EventCard/EventCard.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+
+import S from './EventCard';
+
+const EventCard = ({ data, theme }) => {
+  const { id, title, time, image, token, quantity } = data;
+
+  return (
+    <S.EventCard theme={theme} key={id}>
```

Comment:
```suggestion
    <S.EventCard>
```
key porp은 이미 부모에서 사용하고 있고
theme이라는 prop은 사용하지 않고 있어서 이 부분은 불필요한 부분이라고 보여지네요!